pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -423,3 +415,26 @@ func getCRICadvisorStats(ca cadvisor.Interface) (map[string]cadvisorapiv2.Contai
 	}
 	return stats, nil
 }
+
+func getContainerLogStats(path string) *statsapi.FsStats {
```

Comment:
Add `TODO`, cache this in container log manager.

Reply:
done